scripts/jenkins/Makefile.jenkins

```diff
@@ -61,8 +61,8 @@ lookup-fault-tolerance-tests: init-lookup
 	find h2o-py/tests -name '*fault_tolerance*.py' -exec basename {} \; >>h2o-py/tests/.lookup.txt
 
 lookup-explain-tests: init-lookup
-	find h2o-r/tests -name '*explain*.R' -exec basename {} \; >>h2o-r/tests/.lookup.txt
-	find h2o-py/tests -name '*explain*.py' -exec basename {} \; >>h2o-py/tests/.lookup.txt
+	find h2o-r/tests/testdir_misc/explain -name '*.R' -exec basename {} \; | grep -v NOPASS >>h2o-r/tests/.lookup.txt
+	find h2o-py/tests/testdir_misc/explain -name '*.py' -exec basename {} \; | grep -v NOPASS >>h2o-py/tests/.lookup.txt
```

Comment:
The `.lookup.txt` serves as test list in the `run.py`. When the test is in the test list we run it even if it is `NOPASS`. Since the `SHAP_NOPASS`  can take a day to finish I skip it here.

When we run `pyunit-small` etc. the `.lookup.txt` serves as exclude list  -  `run.py` excludes those tests and then finds the remaining tests while skipping the `NOPASS` tests. (https://github.com/h2oai/h2o-3/blob/1e4e8cad70fd9a27b6de7bdd8ecddea36274c37c/scripts/run.py#L1344-L1347)